scripts/diff-features.js

```diff
@@ -0,0 +1,97 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+const yargs = require('yargs');
+
+function main({ ref1, ref2, format }) {
```

Comment:
It occurs to me now that we could decompose this further, to help other use cases better by separating the checkout-and-enumeration stuff from the feature set diffing stuff. Perhaps this script ought to take two JSON files arguments, instead of refs, then you could do something like:

```shell
$ node ./scripts/diff-features.js <(command-that-enumerates-a-side) <(command-that-enumerates-b-side)
```

(The checkout stuff could move to `enumerate-features`, as an option.)